### DISCUSSION TITLE
Cat food

### DISCUSSION THREAD

**Discussion by CmckXiaoguai** [vectorwing/FarmersDelight#1040] [Ideas]
This mod is brilliant! It is one of the best mods I have ever seen. I appreciate the effort the development teams and developers have put into it. Just a little suggestion: Could you add cat food to the mod for cat lovers like me? Thank you so much. Even not that necessary.

**Comment by MarioSMB** [comment]
This could potentially fit into the gameplay mechanics a little actually; cats have a chance to leave a "gift" at your bed after sleeping, so if you feed a cat it could guarantee a "gift" the following day.
